crates/concolor/src/color/mod.rs

```diff
@@ -120,12 +125,24 @@ fn init() -> usize {
             flags |= InternalFlags::TTY_STDERR;
         }
     }
+    #[cfg(not(feature = "interactive"))]
+    {
+        // Default to assuming the stream is interactive (or the user doesn't care)
+        flags |= InternalFlags::TTY_STDOUT;
+        flags |= InternalFlags::TTY_STDERR;
+    }
 
     #[cfg(feature = "windows")]
     if concolor_query::windows::enable_ansi_colors().unwrap_or(false) {
         flags |= InternalFlags::ANSI_WIN;
     }
 
+    #[cfg(not(any(feature = "term", feature = "windows")))]
+    {
+        // Default to assuming the terminal supports ANSI (or the user doesn't care)
+        flags |= InternalFlags::ANSI_SUPPORT;
+    }
```

Comment:
The reason I put this here is because doing it the way you suggest, if `feature = "windows"` but `not(feature = "term")`, ANSI will always be enabled, which is the bug you are referring to. Currently there is a different bug where ANSI is never enabled if both `not(feature = "windows")` and `not(feature = "term")`.